pkg/ddl/ingest/backend_mgr.go

```diff
@@ -106,31 +133,106 @@ func (m *litBackendCtxMgr) Register(
 	if !ok {
 		return nil, genBackendAllocMemFailedErr(ctx, m.memRoot, jobID)
 	}
-	cfg, err := genConfig(ctx, m.memRoot, jobID, unique, resourceGroupName)
+	cfg, err := genConfig(ctx, m.encodeJobSortPath(jobID), m.memRoot, unique, resourceGroupName)
```

Comment:
i suggest one day we remove dependency on lightning.Config, and depends on local.BackendConfig directly

Reply:
`litBackendCtx` still uses `*lightning.Config`, I'll leave it for future PR